drizzle-orm/package.json

```diff
@@ -174,7 +174,7 @@
 		"@prisma/client": "5.14.0",
 		"@tidbcloud/serverless": "^0.1.1",
 		"@types/better-sqlite3": "^7.6.4",
-		"@types/node": "^20.2.5",
+		"@types/node": "^22.9.1",
```

Comment:
This is required to reference node:sqlite but may cause backwards compatibility issues for the node APIs in this repository.